app/server/methods/data.js

```diff
@@ -1,10 +1,37 @@
+function _exportDataToConcernedColection(collectionName, data) {
+  if (!Array.isArray(data)) throw "Data not in required format";
+
+  let collections = {
```

Comment:
This should be `const`, since we are not changing the reference. Default to `const`.